deadlock_examples/Account.rb

```diff
@@ -0,0 +1,29 @@
+class Account 
+  include Celluloid
+
+  def initialize
+    @mutex = Mutex.new
+    @balance = 500
+  end
+
+  def withdraw(amount)
+   @balance = @balance - amount;
+ end
+
+ def deposit(amount)
+   @balance = @balance + amount;
+ end
+
+ def transfer(from, to, amount)
+  @mutex.lock
```

Comment:
Just curious, why not use `synchronize {}` rather than explicitly `lock` and `unlock`?